samples/compiler_plugins/xnnpack/src/xnnpack_sample/IR/XnnpackOps.td

```diff
@@ -18,7 +18,7 @@ class Xnnpack_Op<string mnemonic, list<Trait> traits = []> :
     Op<Xnnpack_Dialect, mnemonic, traits> {
 }
 
-def Xnnpack_Mul2Op : Xnnpack_Op<"mul2", []> {
+def Xnnpack_Multiply2Op : Xnnpack_Op<"multiply_2", []> {
```

Comment:
nit: we can drop `_2` since multiply is usually a binary op.

Reply:
Yeah, I also don't like the `2`. However, I'm trying to [match the naming](https://github.com/google/XNNPACK/blob/35775027ed07e4425968a62cae4814934b6b4834/include/xnnpack.h#L927) used by XNNPACK for consistency. (although, looking at XNNPACK again, the name should be `multiply2`).